### ISSUE TITLE
Errata

### ISSUE THREAD

**Issue by mnot** [reschke/xml2rfc#6]
Should the stylesheet only show the errata message if there are _verified_ errata, and should the link be to the list of just verified errata?

It's kind of misleading to see "this document has errata" only to follow the link and see that it's only a rejected errata...

Example:
  http://httpwg.github.io/specs/rfc7234.html#rfc.status

**Comment by reschke** [comment]
True, but I currently do not have this information. Will follow up.

**Comment by reschke** [comment]
To fix this, I'll need a CORS-enabled machine-readable resource that gives me errata information for a given RFC #. I have asked for that many times, but no success.

(<https://www.rfc-editor.org/rfc/rfc7234.json> only states the errata URL, but no details)

Maybe it would help if somebody else asked :-)

**Comment by mnot** [comment]
Perhaps this would be useful to discuss in the RSWG...
